### PR TITLE
C#: Upgrade projects to .net 5

### PR DESCRIPTION
**Pull request by tamasvajk**

### CODE REVIEW

File: csharp/ql/src/semmle/code/cil/ConsistencyChecks.qll [hunk 1]
```diff
@@ -79,7 +79,13 @@ class MissingValue extends InstructionViolation {
  * A call that does not have exactly one `getTarget()`.
  */
 class MissingCallTarget extends InstructionViolation {
-  MissingCallTarget() { exists(Call c | c = instruction | count(c.getTarget()) != 1) }
+  MissingCallTarget() {
+    exists(Call c | c = instruction |
+      count(c.getTarget()) != 1 and not c instanceof Opcodes::Calli
+      or
+      count(c.(Opcodes::Calli).getTargetType()) != 1 and c instanceof Opcodes::Calli
```
Comment (hvitved) [inline csharp/ql/src/semmle/code/cil/ConsistencyChecks.qll:86]: Remove `and c instanceof Opcodes::Calli`

**Review by hvitved — CHANGES_REQUESTED** [review]
Very nice, just one trivial comment.

File: csharp/ql/src/semmle/code/cil/ConsistencyChecks.qll (hunk 1 above)
Comment (hvitved) [reply csharp/ql/src/semmle/code/cil/ConsistencyChecks.qll:86]: D'oh, I had not seen the `count`; you are right in reverting the change, the conjunct is actually needed.

**Comment by tamasvajk** [comment]
@hvitved Could you approve this PR? We can ignore the failing tests here, as they are passing on the corresponding internal PR.

**Review by hvitved — APPROVED** [review]

**Pull request merged**